apps/tokenomics/components/Home/Bonding/BondingList/utils.js

```diff
@@ -0,0 +1,177 @@
+import { ethers } from 'ethers';
+import { gql } from 'graphql-request';
+import { isNumber, isString, memoize } from 'lodash';
+
+import { VM_TYPE, notifyError } from '@autonolas/frontend-library';
+
+import { ADDRESSES } from 'common-util/constants/addresses';
+import { ADDRESS_ZERO } from 'common-util/constants/numbers';
+import { DEX } from 'common-util/enums';
+import { AUTONOLAS_GRAPH_CLIENTS } from 'common-util/graphql/clients';
+
+export const getProductValueFromEvent = (product, events, keyName) => {
+  if ((events || []).length === 0) {
+    return product[keyName];
+  }
+
+  if (product.token !== ADDRESS_ZERO) {
+    return product[keyName];
+  }
+
+  const event = events?.find((e) => e?.productId === `${product.id}`);
+  if (!event) notifyError('Product not found in the event list');
+  return event[keyName];
+};
+
+/**
+ *
+ * @param {Number | String} lpTokenValue
+ * @param {Number | String} discount
+ */
+export const getLpTokenWithDiscount = (lpTokenValue, discount) => {
+  const priceInBn = ethers.toBigInt(lpTokenValue);
```

Comment:
do you think `toBigInt` still exists in ethers v6? https://docs.ethers.org/v6/migrating/#migrate-bigint

Reply:
great thanks for some reason it wasn't found through the search